tests/integration/test.compaction.js

```diff
@@ -507,6 +507,18 @@ adapters.forEach(function (adapter) {
       });
     });
 
+    it('#8525 - Only compact document with seq > last_seq', async function () {
```

Comment:
To skip tests for remote databases, you could call skip on `this`.
E.g:
```js
// check for remote target:
if(dbs.name.startsWith("http")) {
    // skip the test:
    this.skip()
}

Reply:
Sry, my code was just meant as an example.
Please add a return statement after the `this.skip();` line, so it exits early.

The current implementation unnecessarily runs the rest of the test function even in the remote case.

Reply:
OK done!
Thanks for the help on the subject.

Reply:
Nothing to thanx... everyone get stuck from time to time.

So my brain tricked me sometimes...
I've just looked up the implementation of Mocha's .skip(), and it turns out:
The return stmt isn't required, because .skip() throws an internal object (this way, .skip always exits the runner).
But it also the didn't hurt, to have a keyword in the line. (Keep praising the syntax highlighter 😁)